shoes-core/lib/shoes/link.rb

```diff
@@ -18,9 +18,9 @@ def initialize(my_app, texts, styles = {}, blk = nil)
       super texts, @style
     end
 
-    # Force hovering evaluation up in parent where we have actual dimensions
-    def eval_in_parent?
-      true
+    # Hover must use containing text block since we're missing dimensions
+    def target
+      text_block
     end
```

Comment:
won't that be a problem though? I.e. don't we need dimensions? Which, I know, is  a grueling thought but if we have a text block that goes link, no link, link won't we trigger hover for all of them? Not in this PR as it's a fix but in general and in the future?